content/docs/buildpack-author-guide/caching-strategies.md

```diff
@@ -0,0 +1,65 @@
+2. Create an empty layer, and
+3. Set `Launch = true`.
+
+This will guarantee that the previously published application image layer in the registry is re-used if the layer metadata matches the requested metadata.  In this most straightforward use-case `Launch` is `true` and both `Build` and `Cache` are set to `false`.  That is to say, the most common case is where `Cache = false`, `Build = false` and `Launch = true`.  It is important to note that the layer is re-used on the OCI registry.  This avoids expensive rebuilds of the layer and expensive pulls of the layer to the host running the build.
+
+Setting `Build = true` makes a layer available to subsequent buildpacks.  Therefore binaries installed to the `bin` directory on a `Build = true` layer are available to subsequent buildpacks during the build phase.  It is also the case that `lib` directories on a `Build = true` later are added to the `LD_LIBRARY_PATH` during the build phase of subsequent buildpacks.  Environment variables defined in a `Build = true` layer are similarly available.  For any layer where `Launch = true` and `Build = true`, a launch layer from the OCI registry can no longer be reused. Instead, the layer must be made available locally so that subsequent buildpacks can use it.
+
+Setting `Cache = true` allows additional fine-grained control over caching.  The `Cache = true` flag caches a layer and allows a buildpack to make _content_ level decisions about the validity of the cache (as opposed to using the less granular metadata).  As an example, suppose a layer where `Launch = true` installs a `jq` binary with version `1.5` and sets `version=1.5` in the layer metadata.  By default, this layer will not be re-used from the registry when a buildpack requests `jq` with `version=1.6` to be installed.  However, setting `Cache = true` makes a previously built layer available during the build.  A buildpack could then prefer to implement logic to restore `jq` with `version=1.5` instead of performing a potentially expensive download of `jq` with `version=1.6`.  The `Cache = true` setting allows for cache validation decisions to be made at a level of granularity that is much finer grained than layer metadata.
+
+Setting `Cache = false, Build = false, Launch = true` is the most common configuration.  If `Cache = false, Build = false, Launch = true` is not appropriate for your layer, then `Cache = true, Build = true, Launch = true` should be the next combination to evaluate:
+
+* When `Cache = true, Build = true, Launch = true`, explicitly setting `Build = true` makes the layer available, to subsequent buildpacks, during the build phase.  As `Cache = true` the layer is restored from local cache before proceeding to the build phase.  For example, a Python distribution could be provided in a cached, build and launch layer. The build phase could verify that the restored cached version of the Python distribution contains Python 3.10 but disregard the patch number of the Python interpreter.
```

Comment:
> explicitly setting `Build = true` makes the layer available

this is true no matter what though. Do we need to restate this?

Reply:
I think so.  Both because repetition can be good and this paragraph talks about the interaction between `build = true` and `cache = true`.  But, I'll happily change it if it's something you feel shouldn't be repeated.